### PR TITLE
Pubsub v2

### PR DESCRIPTION
**Pull request by deni-yoncheva**
- changed the main function to work with push

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+*.csv
+*.json
+levenstein_distance.py
+scraperapi_v1.py
+test_levenstein_api.py
```
Comment (saltas888) [inline .gitignore:5]: It's weird to have source code files in gitignore. If we don't need them, just delete them. If they are just in your machine, do not commit them.
This is dangerous in the future if someone else names like that his new file, then it won't get commit.

File: pub_v1.py [hunk 2]
```diff
@@ -25,27 +25,32 @@ def callback(f):
...
+        index = 0
+        for row_index, row in enumerate(a):
             company_id = row['id']
             name = row['NAME']
             data = {
-                "jobID":"",
                 "method":"url",
-                "search_item":name
+                "search_item":name,
+                "url_score":-100,
```
Comment (saltas888) [inline pub_v1.py:44]: it'd be good to leave a comment to explain what is this -100

File: main.py [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+        final_json = {
+            "name":name,
+            "website_url":max_res,
+            "url_score":-1
+            }
+        
+    return final_json
+
+# Run the whole thing
+def do_sth_with_message(message):
```
Comment (saltas888) [inline main.py:320]: That's the best name for a function I've ever seen 🤣 .  You can just delete the [main definition](https://github.com/dealroom/scraperapi/pull/2/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR343-R349) and name this as `main`

**Review by saltas888 — COMMENTED** [review]
So, main and sub.py are very similar. Why do we need them both?

---

I left a few comments, would be good if you have the time to check them. Otherwise ignore.

File: main.py (hunk 3 above)
Comment (deni-yoncheva) [reply main.py:320]: i have to say i got this from some tutorial :D

**Pull request merged**